docker/init.sql

```diff
@@ -0,0 +1 @@
+create database coupon if not exists coupon;
```

Comment:
```suggestion
create database if not exists coupon;
```

https://dev.mysql.com/doc/refman/8.4/en/create-database.html

Reply:
앗... ㅎㅎ;; 뭐야 이건...ㅎㅎ;;